client/components/jetpack/jetpack-product-info/regular-list.tsx

```diff
@@ -1,14 +1,55 @@
-import { FunctionComponent, ReactNode } from 'react';
+import { useRef, useEffect, type FC, type ReactNode } from 'react';
 
 type Props = {
 	items: ReactNode[];
 };
 
-const JetpackProductInfoRegularList: FunctionComponent< Props > = ( { items } ) => {
+const JetpackProductInfoRegularList: FC< Props > = ( { items } ) => {
+	const listRef = useRef< HTMLUListElement | null >( null );
+
+	const highlightListItem = ( target: HTMLElement ) => {
+		target.animate(
+			[ { backgroundColor: 'var(--studio-yellow-5)' }, { backgroundColor: 'initial' } ],
+			{
+				duration: 500,
+				easing: 'ease-out',
+			}
+		);
+	};
+
+	// Highlight list item when its content changes
+	useEffect( () => {
+		if ( ! listRef.current ) {
+			return;
+		}
+
+		const observer = new MutationObserver( ( record ) => {
+			for ( const mutation of record ) {
+				if ( mutation.type !== 'characterData' ) {
+					continue;
+				}
+
+				const targetParent = mutation.target.parentNode as HTMLElement | null;
+				if ( targetParent?.className === 'jetpack-product-info__regular-list-item' ) {
```

Comment:
Is there a chance that the element won't have that class? From what I see, we're always adding that classname to the `li` elements?

Reply:
This check is probably not absolutely necessary here, the main reason I had it there was in case the list is mutated in other ways (maybe some code later on is added to mutate the list item itself and then the parent item is the '<ul>' element)

I just wanted to make sure we didn't end up animating something on accident later on

Reply:
Fair enough. No harm in keeping the check just in case.